src/ServiceControl.Audit.Persistence.RavenDb5/RavenDbPersistenceConfiguration.cs

```diff
@@ -97,11 +96,11 @@ static int GetExpirationProcessTimerInSeconds(PersistenceSettings settings)
 
         const int ExpirationProcessTimerInSecondsDefault = 600;
 
-        internal const string DatabaseNameKey = "ServiceControl/Audit/RavenDb5/DatabaseName";
-        internal const string DatabasePathKey = "ServiceControl.Audit/DbPath";
-        internal const string ConnectionStringKey = "ServiceControl/Audit/RavenDb5/ConnectionString";
-        internal const string HostNameKey = "ServiceControl.Audit/HostName";
-        internal const string DatabaseMaintenancePortKey = "ServiceControl.Audit/DatabaseMaintenancePort";
-        internal const string ExpirationProcessTimerInSecondsKey = "ServiceControl.Audit/ExpirationProcessTimerInSeconds";
+        public const string DatabaseNameKey = "ServiceControl/Audit/RavenDb5/DatabaseName";
+        public const string DatabasePathKey = "ServiceControl.Audit/DbPath";
+        public const string ConnectionStringKey = "ServiceControl/Audit/RavenDb5/ConnectionString";
+        public const string HostNameKey = "ServiceControl.Audit/HostName";
+        public const string DatabaseMaintenancePortKey = "ServiceControl.Audit/DatabaseMaintenancePort";
+        public const string ExpirationProcessTimerInSecondsKey = "ServiceControl.Audit/ExpirationProcessTimerInSeconds";
```

Comment:
Is the reason for these to be public to use them in tests? If that’s the case can we use an `InternarlsVisibleTo` attribute?

Reply:
The acceptance tests use them so I was unsure if that was a good idea?

I think this is good enough for now since we don't expose this as a public API